src/Connections/FeedConnection.ts

```diff
@@ -28,6 +33,65 @@ export class FeedConnection extends BaseConnection implements IConnection {
         return new FeedConnection(roomId, event.stateKey, event.content, config.feeds, as, storage);
     }
 
+    static async validateUrl(url: string): Promise<void> {
+        try {
+            new URL(url);
+            const res = await axios.get(url);
```

Comment:
*Thought*: Can we save some bytes by doing a `HEAD` request?

Semantically it makes more sense, and we could fallback to a GET if we don't like the response? (I'd imagine some web servers serve dumb results when you ask for HEAD)

Reply:
Good idea: done in ac32228e785bb415b4a0760e44b991957ee75f76